vm/factory/systemSCFactory.go

```diff
@@ -289,6 +289,21 @@ func (scf *systemSCFactory) Create() (vm.SystemSCContainer, error) {
 		return nil, err
 	}
 
+	err = scf.systemEI.SetSystemSCContainer(scf.systemSCsContainer)
```

Comment:
this is so hard coded for our own genesis

Reply:
sadly, yes. That is why the next system SCs will need to make their trie changes only when they get activated.